spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/it/PubSubTemplateIntegrationTests.java

```diff
@@ -16,10 +16,9 @@
 
 package org.springframework.cloud.gcp.autoconfigure.pubsub.it;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
No asterisk imports, please. 
Seems for me we have a Checkstyle rule on the matter.
Otherwise we need to add it.